### PR TITLE
Add note for brats18 data access

### PR DESCRIPTION
**Pull request by ZiyueXu77**
BraTS18 data link broken, add a note to readme. Also bug fix for prostate.

### CODE REVIEW

File: examples/prostate/prostate_2D/job_configs/prostate_fedavg/app/config/config_fed_client.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
   "components": [
     {
       "id": "prostate-learner",
-      "path": "pt.learners.supervised_monai_prostate_learner.SupervisedMonaiProstateLearner",
```
Comment (holgerroth) [inline examples/prostate/prostate_2D/job_configs/prostate_fedavg/app/config/config_fed_client.json:27]: I think `pt.` is required if the script is started from the `brats18` folder and the PYTHONPATH is set as [here](https://github.com/NVIDIA/NVFlare/blob/9a1b226bfb507fce8ea5a1cf3fa15c2a00946806/examples/brats18/start_fl_poc.sh#L3).

**Review by holgerroth — CHANGES_REQUESTED** [review]
I think we need to keep the "pt." in the path.

File: examples/prostate/prostate_2D/job_configs/prostate_fedavg/app/config/config_fed_client.json (hunk 1 above)
Comment (ZiyueXu77) [reply examples/prostate/prostate_2D/job_configs/prostate_fedavg/app/config/config_fed_client.json:27]: in past version these were put under pt/, but recent version we changed to custom/ , so need to removed pt. to make the path recognizable.

File: examples/brats18/README.md [hunk 2]
```diff
@@ -6,6 +6,8 @@ This example shows how to use [NVIDIA FLARE](https://nvflare.readthedocs.io/en/m
 It uses [MONAI](https://github.com/Project-MONAI/MONAI),
 which is a PyTorch-based, open-source framework for deep learning in healthcare imaging, part of the PyTorch Ecosystem.
 ### BraTS
+#### Note: currently the official BraTS 2018 web page is off-line, hence the official data resource is inaccessible, we will update the link once we receive an update.
```
[the remark this answers is not in the document]
Comment (ZiyueXu77) [reply examples/brats18/README.md]: brats back on line

**Comment by ZiyueXu77** [comment]
brats back online

File: examples/advanced/prostate/prostate_2D/job_configs/prostate_fedavg/app/config/config_fed_client.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
   "components": [
     {
       "id": "prostate-learner",
-      "path": "pt.learners.supervised_monai_prostate_learner.SupervisedMonaiProstateLearner",
```
Comment (holgerroth) [inline examples/advanced/prostate/prostate_2D/job_configs/prostate_fedavg/app/config/config_fed_client.json:27]: Is this fixing a bug?

Comment (ZiyueXu77) [reply examples/advanced/prostate/prostate_2D/job_configs/prostate_fedavg/app/config/config_fed_client.json:27]: yes, pt no longer works for the current setting

**Review by holgerroth — APPROVED** [review]

**Pull request merged**